features/fixtures/simple_app/spec/spec_helper.rb

```diff
@@ -7,7 +7,9 @@
 require_relative '../../../../lib/crystalball/rails'
 Crystalball::MapGenerator.start! do |c|
   c.register Crystalball::MapGenerator::CoverageStrategy.new
-  c.register Crystalball::MapGenerator::AllocatedObjectsStrategy.build(only: ['Object'])
+  allocated_strategy = Crystalball::MapGenerator::AllocatedObjectsStrategy.build(only: ['Object'])
```

Comment:
Why are you creating this variable? It does not make it clearer

Reply:
yeah. no need for this variable at all